src/components/application_manager/src/commands/hmi/request_to_hmi.cc

```diff
@@ -56,6 +56,21 @@ bool ChangeInterfaceState(ApplicationManager& application_manager,
                                                            interface_state);
     return is_available;
   }
+  if (response_from_hmi[strings::params].keyExists(hmi_response::code)) {
+    int32_t response_code_ =
+        response_from_hmi[strings::params][hmi_response::code].asInt();
+    if (response_code_ !=
+            static_cast<uint32_t>(hmi_apis::Common_Result::SUCCESS) ||
+        response_code_ !=
+            static_cast<uint32_t>(hmi_apis::Common_Result::WARNINGS)) {
```

Comment:
@ValeriiMalkov is it possible to store response_code as hmi_apis::Common_Result to avoid multiple static casts?

Reply:
@LuxoftAKutsan That was fixed and force updated. See commit 423b389c93a5625b3bd4b27c90b68dc134877e7b